.configure.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+if [[ "${FATS_CONFIGURED:-x}" == "x" ]]; then
+  echo -e "Configuring FATS run:"
+  echo -e "    cluster ${ANSI_BLUE}${CLUSTER}${ANSI_RESET}"
+  echo -e "    registry ${ANSI_BLUE}${REGISTRY}${ANSI_RESET}"
+
+  source `dirname "${BASH_SOURCE[0]}"`/clusters/${CLUSTER}/configure.sh
```

Comment:
nit: `.` is POSIX compliant, `source` is not and afaik, they do the same things. Maybe replace all `source` to `.` ?

Reply:
I tend to prefer `source` as it's more explicit. If we ever need to run in an environment that doesn't support source we can replace it with `.`